integration-tests/ibm-cos/pom.xml

```diff
@@ -39,6 +39,18 @@
             <groupId>io.quarkus</groupId>
             <artifactId>quarkus-resteasy</artifactId>
         </dependency>
+        <dependency>
+        	<groupId>org.apache.camel.quarkus</groupId>
+        	<artifactId>camel-quarkus-direct</artifactId>
+        </dependency>
+        <dependency>
+            <groupId>io.quarkus</groupId>
+            <artifactId>quarkus-resteasy-jackson</artifactId>
+        </dependency>
+        <dependency>
+            <groupId>io.quarkus</groupId>
+            <artifactId>quarkus-resteasy-jsonb</artifactId>
+        </dependency>
```

Comment:
Usually you'd pick one JAX-RS JSON serialization library. Either Jackson or jsonb.

I'd be interested to see what happens if you use only jsonb. Will the native tests still work? It might indicate that you need to add `quarkus-jackson` to the `ibm-cos` extension.